docker-compose.yaml

```diff
@@ -60,6 +60,8 @@ services:
       - "com.centurylinklabs.watchtower.enable=true"
     ports:
       - "10009:10009"
+  nighthawk-lg:
+    image: envoyproxy/nighthawk-dev:latest
```

Comment:
Curious, what's the size of this image?

Reply:
Use of the `-dev` is temporary, until when?

After `-dev` is gone, what is our strategy for pinning to a specific version? This question is the same for other generators.

Reply:
42 mb

Reply:
> Use of the -dev is temporary, until when?

Until nighthawk makes its v1.0 debut.

> After -dev is gone, what is our strategy for pinning to a specific version? This question is the same for other generators.

This may get addressed when we deduce the strategy of versioning the adapters and pinning. Most probably, keep up with the latest as Load Generators may only get better with their new versions with very few breaking changes but this may not be same when we address the adapters.